test/python/scenarios.json

```diff
@@ -48,6 +48,15 @@
             }
         }
     },
```

Comment:
Can you temporarily remove [centos 7](https://github.com/devcontainers/features/blob/main/test/python/scenarios.json#L206-L213) test? It's currently been fixed in another PR. 

Just want to make sure all other tests are passing, thank you!

Reply:
Done